forms/session.go

```diff
@@ -0,0 +1,58 @@
+package forms
+
+import (
+	"go-google-scraper-challenge/helpers"
+	"go-google-scraper-challenge/models"
+	"log"
+
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type SessionForm struct {
+	Email    string `form:"email" valid:"Email; Required"`
+	Password string `form:"password" valid:"Required;"`
+}
+
+var currentUser *models.User
+
+// Valid adds custom validation to registration form, sets error when the validation failed.
+func (sessionForm *SessionForm) Valid(v *validation.Validation) {
+	user, err := models.GetUserByEmail(sessionForm.Email)
+	if err != nil {
+		validationError := v.SetError("Email", "Incorrect email or password")
```

Comment:
According to our [convention](https://github.com/nimblehq/compass/blob/development/_docs/development/code-conventions/go.md#errors)
```suggestion
		err := v.SetError("Email", "Incorrect email or password")
```

and also in the `else` statement.

Reply:
@gutakk fixed on [a19164e](https://github.com/carryall/go-google-scraper-challenge/pull/31/commits/a19164e2bb06189e0389fc079ecb679c6c760e66)